### ISSUE TITLE
sandboxing fails for 32-bit Intel

### ISSUE THREAD

**Issue by andreasbaumann** [universal-ctags/ctags#1937]
Tested on 32-bit Intel, Archlinux32.

The following tests are failing:

```
Failed tests
============================================================
sandbox-default-req/stdout-compare
sandbox-default-req/stderr-compare
sandbox-default-req/exit-compare
sandbox-with-eager-guessing/stdout-compare
sandbox-with-eager-guessing/stderr-compare
sandbox-with-eager-guessing/exit-compare
sandbox/stdout-compare
sandbox/stderr-compare
sandbox/exit-compare
```

A single execution of:

```
 {     echo '{"command":"generate-tags", "filename":"input.unknown", "size": 44}';     echo '/* -*- c -*- */ int main(void) { return 0; }'; } | ./ctags --options=NONE  -G --_interactive=sandbox 
```

results in:

```
Bad system call         (core dumped) 
```

There are some seccomp_rule_add missing for the 32-bit kernel.
(Pull request follows)

**Comment by andreasbaumann** [comment]
Pull request is #1938 .

**Comment by masatake** [comment]
Fixed by #1938.

**Comment by masatake** [comment]
Thank you.
